Server/VikingVault.API/Startup.cs

```diff
@@ -69,7 +69,7 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env, VikingVaultDbContext context)
         {
-	        context.Database.Migrate();
+	        //context.Database.Migrate();
```

Comment:
Delete this line of commented code if you don't need it

Reply:
That line shouldn't be commented. I removed the comment.